agent/php_txn.c

```diff
@@ -1109,3 +1109,93 @@ nr_status_t nr_php_txn_end(int ignoretxn, int in_post_deactivate TSRMLS_DC) {
 
   return NR_SUCCESS;
 }
+
+extern void nr_php_txn_add_code_level_metrics(
+    nr_attributes_t* attributes,
+    const nr_php_execute_metadata_t* metadata) {
+#if ZEND_MODULE_API_NO < ZEND_7_0_X_API_NO /* PHP7+ */
+  (void)attributes;
+  (void)metadata;
+  return;
+}
+#else
+  /*
+   * Check if code level metrics are enabled in the ini.
+   * If they aren't, exit and don't add any attributes.
+   */
+  if (!NRINI(code_level_metrics_enabled)) {
+    return;
+  }
+
+  /* Current CLM functionality only works with PHP 7+ */
```

Comment:
This comment seems to be in the wrong place

Reply:
Please see [fdbf4a2](https://github.com/newrelic/newrelic-php-agent/pull/585/commits/fdbf4a2df8224ef2425a5e6a56291f374f33a645)